apps/files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -106,7 +108,7 @@ export default {
     },
 
     $_ocCollaborators_hasAdditionalPermissions() {
-      if (this.selectedRole && this.selectedRole.name !== 'advancedRole') {
+      if (this.selectedRole && this.selectedRole.additionalPermissions) {
```

Comment:
The body of this new if statement is always `true`. So you could simplify the whole function body to `return this.selectedRole && this.selectedRole.additionalPermissions`.